packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.tsx

```diff
@@ -133,6 +136,11 @@ export class PaymentProvisioning {
     this.errorMessage = e.detail.message;
   }
 
+  setBusinessPaymentVolume = (e: CustomEvent) => {
+    let business = e.detail.data.data;
```

Comment:
Is the `justifi-additional-questions-form-step` `onSubmitted` event wrapping the event payload in `{ data: ... }`? If it weren't, this could just be `e.detail.data`

Reply:
Ok, I checked on this and it is indeed emitting it by assigning the event payload to `data`

I have it setup using a type I made for the submit events in the BusinessForm components. 

```
export interface BusinessFormSubmitEvent {
  data: any;
  metadata?: BusinessFormEventMetaData;
}
```

So, this can be adjusted, but would need to adjust a handful of additional files. I'll save this potentially for the next refactor goals.